src/dgc_cert.rs

```diff
@@ -6,7 +6,9 @@ pub struct DgcCertName {
     #[serde(default)]
     #[serde(skip_serializing_if = "Option::is_none")]
     pub gn: Option<String>,
-    pub r#fn: String,
+    #[serde(default)]
```

Comment:
FYI: `#[serde(default)]` is not needed if you have `skip_serializing_if = "Option::is_none"`.